tools/openvino_dev/setup.py

```diff
@@ -55,6 +55,11 @@
         'extract_entry_points': True,
         'extract_extras': True,
     },
+    "openmodelzoo-modelapi": {
+        'src_dir': OPENVINO_DIR / 'tools' / 'pot' / 'thirdparty' / 'open_model_zoo' / 'demos' / 'common' / 'python', # noqa:E501
```

Comment:
Why are OMZ tools (and now API) in tools/pot/thirdparty/open_model_zoo? It seems very strange to me. OMZ is not primarily a thirdparty tool used by POT.